c10/core/SymIntNodeImpl.cpp

```diff
@@ -0,0 +1,13 @@
+#include <c10/core/SymInt.h>
+#include <c10/core/SymIntNodeImpl.h>
+
+namespace c10 {
+
+c10::SymInt SymIntNodeImpl::toSymInt() {
+  // We will need to figure out a way
+  // to dedup nodes
```

Comment:
nodes means SymIntNodeImpl here?

Reply:
Actually I think this comment is obsolete now